ionoscloud/resource_vpn_wireguard_gateway_test.go

```diff
@@ -177,7 +177,7 @@ resource` + ` ` + constant.WireGuardGatewayResource + ` ` + constant.WireGuardGa
   description = "description1"
   gateway_ip = ` + constant.IpBlockResource + `.` + constant.IpBlockTestResource + `.ips[0]
   interface_ipv4_cidr =  "192.168.1.101/24"
-  private_key = "private"
+  private_key = "0HpE4BNwGHabeaC4aY/GFxB6fBSc0d49Db0qAzRVSVc="
```

Comment:
dummy key right?

Reply:
@cristiGuranIonos yes, this key is also used in the [API documentation](https://ionos-cloud.github.io/rest-api/docs/vpn/v1.ea/#tag/WireguardGateways/operation/wireguardgatewaysPost).